single_step_avd/src/merkle_tree_avd/mod.rs

```diff
@@ -422,16 +425,20 @@ mod tests {
     impl MerkleTreeAVDParameters for MerkleTreeAVDTestParameters {
         const MAX_UPDATE_BATCH_SIZE: u64 = 8;
         const MAX_OPEN_ADDRESSING_PROBES: u8 = 2;
-        type MerkleTreeParameters = MerkleTreeTestParameters;
+        type SMTStorer = SMTMemStore<MerkleTreeTestParameters>;
     }
 
     type TestMerkleTreeAVD = MerkleTreeAVD<MerkleTreeAVDTestParameters>;
 
     #[test]
     fn update_and_verify_test() {
-        let mut rng = StdRng::seed_from_u64(0_u64);
+        let mut rng = StdRng::seed_from_u64(0u64);
         let crh_parameters = H::setup(&mut rng).unwrap();
-        let mut avd = TestMerkleTreeAVD::new(&mut rng, &crh_parameters).unwrap();
+        // let smt_mem_store: SMTMemStore<MerkleTreeTestParameters> = SMTMemStore::new(&[0u8; 16], &crh_parameters).unwrap();
+        let mtavd_mem_store: MTAVDMemStore<MerkleTreeAVDTestParameters> = MTAVDMemStore::new(&[0u8; 16], &crh_parameters).unwrap();
```

Comment:
@nirvantyagi also looks same:

the function or associated item `new` exists for struct `merkle_tree_avd::store::mem_store::MTAVDMemStore<_>`, but its trait bounds were not satisfied function or associated item cannot be called on `merkle_tree_avd::store::mem_store::MTAVDMemStore<_>` due to unsatisfied trait bounds note: the following trait bounds were not satisfied: `merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::digest::Update` which is required by `merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::Digest` `merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::digest::FixedOutput` which is required by `merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::Digest` `merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::digest::Reset` which is required by `merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::Digest` `merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: std::clone::Clone` which is required by `merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::Digest` `merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: std::default::Default` which is required by `merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::Digest` `&merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::digest::Update` which is required by `&merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::Digest` `&merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::digest::FixedOutput` which is required by `&merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::Digest` `&merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::digest::Reset` which is required by `&merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::Digest` `&merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: std::default::Default` which is required by `&merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::Digest` `&mut merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::digest::Update` which is required by `&mut merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::Digest` `&mut merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::digest::FixedOutput` which is required by `&mut merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::Digest` `&mut merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::digest::Reset` which is required by `&mut merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::Digest` `&mut merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: std::clone::Clone` which is required by `&mut merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::Digest` `&mut merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: std::default::Default` which is required by `&mut merkle_tree_avd::store::mem_store::MTAVDMemStore<_>: sha3::Digest` help: items from traits can only be used if the trait is in scope